.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ 'main' ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:        
+  test:
+    runs-on: macos-13
```

Comment:
Could we update this to macos-latest?

Reply:
Github action still defaults `macos-latest` to version `12.x.x` which is missing the latest Xcode https://github.com/actions/runner-images/blob/macOS-12/20240329.1/images/macos/macos-12-Readme.md#xcode

I've switched this to `macos-14` instead. Let me know if this works

Reply:
It's saying it defaults to 14 here. But specifying macOS-14 is fine! Thanks for updating.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Reply:
I think they haven't updated the `latest` images to point to macOS 14, when I tried latest, it should it's macOS 12 https://github.com/cbaker6/ResearchKit/actions/runs/8620787466/job/23628313753#step:1:8